lib/providers/restful-provider.js

```diff
@@ -0,0 +1 @@
+module.exports = function(){};
```

Comment:
Mind just throwing a not implemented error here? Not that it's a big deal, I think it's just a good habit to get into